crates/goose/src/providers/databricks.rs

```diff
@@ -8,11 +8,12 @@ use super::base::{Provider, ProviderUsage, Usage};
 use super::configs::{DatabricksAuth, DatabricksProviderConfig, ModelConfig, ProviderModelConfig};
 use super::model_pricing::{cost, model_pricing_for};
 use super::oauth;
-use super::utils::{
-    check_bedrock_context_length_error, check_openai_context_length_error, get_model,
-    messages_to_openai_spec, openai_response_to_message, tools_to_openai_spec,
-};
+use super::utils::{check_bedrock_context_length_error, get_model};
 use crate::message::Message;
+use crate::providers::openai_utils::{
+    check_openai_context_length_error, messages_to_openai_spec, openai_response_to_message,
+    tools_to_openai_spec,
+};
```

Comment:
nit: we should prob use the get_openai_usage and handle_response in this one as well.

Reply:
done